doc/man1/openssl-verify.pod.in

```diff
@@ -99,9 +99,9 @@ with a B<->.
 
 =item I<certificate> ...
 
-One or more target certificates to verify. If no certificates are given,
-this command will attempt to read a certificate from standard input.
-If a certificate chain has multiple problems, this program attempts to
+One or more target certificates to verify, one per file. If no certificates are
+given, this command will attempt to read a single certificate from standard
+input. If a certificate chain has multiple problems, this program attempts to
```

Comment:
Maybe the 'If a certificate chain ...' line is kind of in the wrong place.
Should it be below the =back??

Would 'leaf' be a better word than 'target'?

Reply:
We use the term 'target' elsewhere in the document. I do not think we should change that in here.